driver-core/src/main/java/com/datastax/driver/core/TableMetadata.java

```diff
@@ -384,12 +385,30 @@ public String exportAsString() {
 
         sb.append(super.exportAsString());
 
-        for (IndexMetadata index : indexes.values()) {
-            sb.append('\n').append(index.asCQLQuery());
+        if (!indexes.isEmpty()) {
+            sb.append('\n');
```

Comment:
We have to be stingy with newlines here because `KeyspaceMetadata` adds one after the tables total metadata, so we don't want duplicate newlines.